CHANGELOG.md

```diff
@@ -18,6 +19,10 @@
 
 - `MlflowNodeHook` have now a before_pipeline_run hook which stores the ProjectContext and enable to retrieve configuration.
 
+### Removed
+
+Give- `kedro mlflow init` command is no longer declaring hooks in `run.py`. You must now [register your hooks manually](docs/source/03_tutorial/02_setup.md#declaring-kedro-mlflow-hooks) in the ``run.py`` (kedro > 0.16.0), ``.kedro.yml`` (kedro >= 0.16.5) or ``pyproject.toml`` (kedro >= 0.16.5)
```

Comment:
~~Give~~
``.kedro.yml`` (kedro >= **0.16.0**)

Reply:
There is no hooks entry in `.kedro.yml` for kedro < 0.16.5